javascript/javascript1/week2/fight_with_gender.js

```diff
@@ -0,0 +1,18 @@
+//we might add a radio button in the html file for the gender get only male or female
+const gender = female;
+function getFullname(gender,firstname, surname, useFormalName)
+{
+    if((useFormalName = true) && (gender = true)){
```

Comment:
There is a difference between `=` and `==`. This `useFormalName = true` will set the value of `useFormalName` to true. You should use two equals `==`.
```suggestion
    if((useFormalName == true) && (gender == true)){
```